kernel_gateway/services/sessions/sessionmanager.py

```diff
@@ -0,0 +1,171 @@
+"""A base class session manager."""
+
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import uuid
+
+from tornado import web
+
+from traitlets.config.configurable import LoggingConfigurable
+from ipython_genutils.py3compat import unicode_type
+
+class SessionManager(LoggingConfigurable):
+
+    def __init__(self, kernel_manager):
+        self.kernel_manager = kernel_manager
+        self._sessions = []
```

Comment:
Sessions are in-memory I see. Any concern about this filling up? Or is the thinking that the KG would be scaled out?